org/invite-collaborator.ts

```diff
@@ -52,30 +55,37 @@ export const inviteCollaborator = wrap(
 `;
 
     try {
-      const invite = await api.orgs.addTeamMembership({
-        // ID of the @gatsbyjs/maintainers team on GitHub
-        id: '1942254',
-        username
-      });
-
-      if (invite.data.state === 'active') {
-        console.log(
-          `@${username} is already a ${invite.data.role} for this team.`
-        );
-      } else {
-        console.log(`We’ve invited @${username} to join this team.`);
+      // This throws if the user isn't a member of the org yet.
+      // If it doesn't throw, it means the user was already invited or has
+      // already accepted the invitation (we ignore the return value here).
+      await api.orgs.checkMembership({ org, username });
+    } catch (error) {
+      // This is a query to check the number of merged pull requests by the user.
+      const query = `org:gatsbyjs+author:${username}+type:pr+is:merged`;
+      const response = await api.search.issues({ q: query });
+      const mergedCount = response.data.total_count;
+      // We only send out an invite on the first PR, in order not to spam users.
+      if (mergedCount === 1) {
```

Comment:
I think we got out of date here — I pushed this change a few hours ago that checks if we've already invited someone:

https://github.com/gatsbyjs/peril-gatsbyjs/blob/master/org/invite-collaborator.ts#L25-L35

In our use-case, I think we need to do that vs. the first-time PR, because we have ~850 existing contributors and over 800 haven't been invited yet. We've got plans to invite our existing contributors with another script, but we don't want to miss anyone.

Do you see any issues with that approach?